db/schema.rb

```diff
@@ -121,6 +121,18 @@
     t.index ["timestamp"], name: "index_alerts_on_timestamp"
   end
 
+  create_table "annotations", charset: "utf8mb4", collation: "utf8mb4_bin", force: :cascade do |t|
+    t.string "name", null: false
+    t.string "value"
+    t.string "annotated_type", null: false
+    t.bigint "annotated_id", null: false
+    t.integer "tenant_id"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["annotated_type", "annotated_id"], name: "index_annotations_on_annotated"
+    t.index ["name"], name: "index_annotations_on_name"
```

Comment:
I wonder if we should better have one index on all 3 - name, annotated_type and annotated_id, but probably name should be the last one because sometimes we want all annotations of an object in which case the first two will be used and when we want a specific annotation, then all 3 will be used.